cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
```

Comment:
And then in the first comment, we can reference it with

    :term:`master services`

and that should link to the Glossary.